docs/usage/post_training_compression/weights_compression/Usage.md

```diff
@@ -527,6 +527,251 @@ Here is the perplexity and accuracy with data-free and data-aware mixed-precisio
     <t/r>
 </table>
 
+Below are the tables showing the accuracy/footprint trade-off for 2 models `Qwen/Qwen2-7B` and
+`microsoft/Phi-3-mini-4k-instruct` compressed with different options.
+
+Footprint overhead is measured by the percentage increase in model size relative to a baseline int4 model, compressed with `ratio=1` and `all_layers=False` options.
+Accuracy metrics are measured on 4 tasks [lambada openai](https://huggingface.co/datasets/EleutherAI/lambada_openai), [wikitext](https://arxiv.org/pdf/1609.07843.pdf),
+[winogrande](https://arxiv.org/abs/1907.10641), [WWB](https://github.com/openvinotoolkit/openvino.genai/tree/master/llm_bench/python/who_what_benchmark/whowhatbench).
+The `average relative error` in the tables below is the mean of relative errors for each of four tasks with respect to
+the metric value for fp32 model. All int4 models are compressed group-wise with `group_size=128` and `mode=CompressionMode.INT4_SYM` and
```

Comment:
Would it be better to use 'mode=CompressionMode.INT4_ASYM' everywhere which is default?

Reply:
There is still some difference in performance between ASYM and SYM on some platforms, that's why it was decided to perform all experiments with SYM.  @AlexKoff88